### ISSUE TITLE
Error codes being read by color blind people

### ISSUE THREAD

**Issue by kathrynKozak** [rstudio/rstudio#4364]
### System details

    RStudio Edition : 
    RStudio Version : 
    OS Version      : 
    R Version       :

**Comment by kathrynKozak** [comment]
The error codes when you type something wrong are always displayed in the color red. I had a color blind student who could not see the color red. I found no way to have the code displayed in a different color.

**Comment by DesiQuintans** [comment]
The colour of the error message depends on the theme you've set in _Tools → Global Options → Appearance_, it just so happens that the default theme, Textmate, has this colour scheme. 

I agree that the default theme should be colourblind-friendly. I have made a theme called [Pebble-safe](https://github.com/DesiQuintans/pebble-themes) using ColorBrewer's colourblind-friendly palettes, but since I'm not colorblind myself, I am unsure how it actually performs.

**Comment by jmcphers** [comment]
@DesiQuintans Great work on the theme! I am colorblind myself, though not in such a way that I can't discern red in most errors. 

@kathrynKozak Any chance you can run @DesiQuintans's theme past your colorblind student? We'd love to have at least one colorblind-friendly theme included in RStudio by default.

**Comment by kathrynKozak** [comment]
I had the student last semester. I will see if I can find him this semester to show him the theme.

Kate

Sent from my iPhone

On Feb 27, 2019, at 3:32 PM, Jonathan <notifications@github.com<mailto:notifications@github.com>> wrote:


@DesiQuintans<https://github.com/DesiQuintans> Great work on the theme! I am colorblind myself, though not in such a way that I can't discern red in most errors.

@kathrynKozak<https://github.com/kathrynKozak> Any chance you can run @DesiQuintans<https://github.com/DesiQuintans>'s theme past your colorblind student? We'd love to have at least one colorblind-friendly theme included in RStudio by default.

**Comment by jmcphers** [comment]
@gtritchie Please triage for the 1.3 accessibility work (punting if necessary)

**Comment by gtritchie** [comment]
See closely related issue: #2574

**Comment by gtritchie** [comment]
Not going to get to this for 1.3; might include a note in the docs referencing availability of Pebble-Safe themes.

**Comment by gtritchie** [comment]
Won't make the bar for 1.4; theme is mentioned in the accessibility documention.
